classifiers/image/CLIPZeroShotClassifier/README.md

```diff
@@ -0,0 +1,3 @@
+# CLIPZeroShotClassifier
```

Comment:
May you please add better documentation and explanation about this executor? We are working hard into improving the READMEs and documentation for the hub executors